Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1218,12 +1218,6 @@ private void LogCommandLineTooLong(SandboxedProcessInfo info)
 
                         if (m_pip.RetryExitCodes.Contains(result.ExitCode) && m_remainingUserRetryCount > 0)
                         {
-                            // Retry if user specifies that the exit code can be retried.
-                            if (await TrySaveAndLogStandardOutputAsync(result) && await TrySaveAndLogStandardErrorAsync(result))
-                            {
-                                await TryLogErrorAsync(result, exitedWithSuccessExitCode);

```

Comment:
>TryLogErrorAsync [](start = 38, length = 16)

Should we log the error details as an info here? So we have telemetry and data on which processes are failing periodically and can investigate?

Reply:
Yes. I also want to address my todo in the test framework and extend this validation to all integration tests. I'll tweak that in a follow up PR.

Reply:
sounds good